src/libcore/slice/mod.rs

```diff
@@ -5521,6 +5521,13 @@ impl<T: PartialOrd> PartialOrd for [T] {
     }
 }
 
+#[unstable(feature = "array_from_single", reason = "newly added", issue = "none")]
```

Comment:
impls are insta-stable.

Reply:
I changed this to `#[stable(feature = "array_from_single", since = "1.42")]`, is that right? I'm not sure what `since` should be or what `feature` means if you don't need `#![feature(...)]` to use the impl.

Reply:
`"1.42.0"` but otherwise OK.